packages/material-ui/src/TextField/TextField.js

```diff
@@ -34,8 +34,8 @@ import Select from '../Select';
  *
  * For advanced cases, please look at the source of TextField by clicking on the
  * "Edit this page" button above. Consider either:
- * - using the upper case props for passing values direct to the components.
- * - using the underlying components directly as shown in the demos.
+ * - using the upper case props for passing values direct to the components
+ * - using the underlying components directly as shown in the demos
  */
```

Comment:
And I thought I was pedantic! 😄 

While we're here though: "passing values directly to"

Reply:
😄 It takes a village to write the docs.
Good catch.  Done.